src/vaishnaviVJun20_ExceptionHandling/Extract.java

```diff
@@ -0,0 +1,44 @@
+package vaishnaviVJun20_ExceptionHandling;
+
+/* Extract ItemId from below input
+
+String input = https://www.facebook.com/technocredits/itemId=12345/
+String input = https://www.facebook.com/technocredits/empid=123&itemId=12345/
+String input = https://www.facebook.com/technocredits/itemId=12345&empid=123/
+String input = https://www.facebook.com/technocredits/empid=123/
+
+Output : 
+item id is 12345 
+        OR
+no item id present. */
+
+public class Extract {
+
+	void getItemIdFromGivenString(String str) {
+		try {
+			int firstindex = str.indexOf("itemId");
+			String temp = str.substring(firstindex);
+			String itemID = "";
+			for (int index = 0; index < temp.length(); index++) {
+				if (Character.isDigit(temp.charAt(index))) {
+					itemID += temp.charAt(index);
+					if (!Character.isDigit(temp.charAt(index + 1))) {
```

Comment:
Suggesting an alternative You can also write : 
try{
String str = input.split("itemId=")[1].replace("/", "").split("&")[0];
System.out.println("Item id is " + str);

}catch(StringIndexOutOfBoundsException siob){
			System.out.println("Item ID -> Not present");
		}

This two line in try block and it will work.